packages/portfolio-api/test/types.test-d.ts

```diff
@@ -0,0 +1,154 @@
+import { expectAssignable, expectNotAssignable, expectType } from 'tsd';
```

Comment:
this is more type tests that I was expecting somehow. I'm not in the habit of maintaining this sort of thing. I might need reminders as things evolve.